akka-actor-tests/src/test/java/akka/actor/JavaAPITestActor.java

```diff
@@ -1,3 +1,7 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
```

Comment:
should this be `2011-2018` in this case? Or do we not care?

Reply:
I think the sbt-header configuration just adding `2018` is OK. We could consider manually updating files we know are older, but I'm OK with the header like this as well. AFAIK the year in a copyright header is mostly there for informational purposes, not really legal ones.